tyk-stack/values.yaml

```diff
@@ -1326,3 +1326,8 @@ tyk-dev-portal:
 
   # podLabels specifies labels to be added in Tyk Developer Portal Pod
   podLabels: {}
+
+  # bootstrapPortal contains information that'd be used to bootsrap dev portal
+  bootstrapPortal:
+    firstName: "admin"
```

Comment:
Will it work without firstName and lastName? Asking to know if this cause any issue to existing users.